cfgov/unprocessed/css/molecules/archived-pill.less

```diff
@@ -0,0 +1,33 @@
+/* topdoc
+  name: Archived pill
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-archived-pill">
+            Archived
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-archived-pill
+  tags:
+    - cfgov-molecules
+*/
+
+.m-archived-pill {
+    padding: unit( 9px / @base-font-size-px, em ) unit( 10px / @base-font-size-px, em );
```

Comment:
Maybe this is as-designed, but the padding looks a little imbalanced to me with these numbers (too much top/bottom padding compared to left/right). You might try `line-height: 1` and the same padding value all around.

Reply:
Updated in https://github.com/cfpb/consumerfinance.gov/pull/6038/commits/6aed924cf92ba897cce6715791d033a41c55bc7f